deltachat-ffi/deltachat.h

```diff
@@ -2097,6 +2101,12 @@ void            dc_stop_ongoing_process      (dc_context_t* context);
  * - DC_QR_TEXT with dc_lot_t::text1=Text
  * - DC_QR_URL with dc_lot_t::text1=URL
  * - DC_QR_ERROR with dc_lot_t::text1=Error string
+ * - DC_QR_WITHDRAW_VERIFYCONTACT, DC_QR_WITHDRAW_VERIFYGROUP
+ *   withdraw own qr-codes, with text1=groupname for groups
+ *   to actually withdraw, call dc_set_config_from_qr() with the code.
+ * - DC_QR_REVIVE_VERIFYCONTACT, DC_QR_REVIVE_VERIFYGROUP,
+ *   revive withdrawn qr-codes, with text1=groupname for groups,
+ *   to actually revive, call dc_set_config_from_qr() with the code.
```

Comment:
Should maybe be DC_QR_*ASK*_WITHDRAW_VERIFYCONTACT and so on? Because, I guess the UI is supposed to ask the user about it?

Reply:
in fact, i had the "ASK" part during development, but it appears to noisy to me :) as also most DC_QR_* constants result in asking the user sth. but do not have "ASK" in their names,  i stroke it :)

Reply:
Currently the docs don't say that the UI is supposed to ask the user in these cases, maybe this hint should be put somewhere else, then? OTOH, that's unrelated to this PR.

I don't think that's a blocker, the reason that I did not approve was that I don't know this code at all and didn't feel confident enough (as written on IRC)

Reply:
you are right, the existing documentation on dc_check_qr() is poor, i tried to improve that a bit, however, i would not go for renaming constants (eg. always add/strike the "ASK").